src/System.Runtime/tests/System/StringTests.netcoreapp.cs

```diff
@@ -33,6 +33,25 @@ public static void Ctor_CharSpan(char[] valueArray, int startIndex, int length,
         }
 
         [Theory]
+        [InlineData("")]
+        [InlineData("a")]
+        [InlineData("this is a test")]
+        [InlineData("\0\u8001\u8002\ufffd\u1234\ud800\udfff")]
+        public static void Create_ReturnsExpectedString(string expected)
```

Comment:
test for null state is OK?
test for length < 0 ?

Reply:
Fixed